contracts/HubPool.sol

```diff
@@ -231,6 +248,33 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
      *                ADMIN FUNCTIONS                *
      *************************************************/
 
+    /**
+     * @notice Pauses the bundle proposal and execution process. This is intended to be used during upgrades or when
+     * something goes awry.
+     * @param pause true if the call is meant to pause the system, false if the call is meant to unpause it.
+     */
+    function setPaused(bool pause) public onlyOwner nonReentrant {
```

Comment:
nit: can we remove `nonReentrant`?

Reply:
if we do then should do the same on all the `onlyOwner` functions.

Reply:
we don't have `nonReeentrant` on any public methods that don't make external calls, including some of the `onlyOwner` functions like `setLiveness`